couchbase/admin/analytics/v1/analytics.proto

```diff
@@ -0,0 +1,216 @@
+syntax = "proto3";
+
+option go_package = "github.com/couchbase/goprotostellar/genproto/admin_analytics_v1;admin_analytics_v1";
+option ruby_package = "Couchbase::Protostellar::Generated::Admin::Analytics::V1";
+option php_namespace = "Couchbase\\Protostellar\\Generated\\Admin\\Analytics\\V1";
+option java_package = "com.couchbase.client.protostellar.admin.analytics.v1";
+option java_multiple_files = true;
+option csharp_namespace = "Couchbase.Protostellar.Admin.Analytics.V1";
+
+import "google/protobuf/timestamp.proto";
+
+package couchbase.admin.analytics.v1;
+
+enum LinkStatus {
+  LINK_STATUS_HEALTHY = 1;
+}
+
+enum CouchbaseEncryptionMode {
+  COUCHBASE_ENCRYPTION_MODE_NONE = 1;
+  COUCHBASE_ENCRYPTION_MODE_HALF = 2;
+  COUCHBASE_ENCRYPTION_MODE_FULL = 3;
+}
+
+service AnalyticsAdminService {
+  rpc CreateLink(CreateLinkRequest) returns (CreateLinkResponse) {}
```

Comment:
Is there a reason why this is missing all the create dataverse/dataset etc... funcs?

Reply:
Creating dataverses/datasets/etc... Is all performed through query itself, rather than through RESTful operations.  Since there is no need for additional logic from CNG, we don't want to provide duplicate identical commands.

Reply:
The same could be said for query index management couldn't it? We have, however, supported that in Protostellar.

Reply:
Query Index Management is a bit different because the query service does not itself provide guarentees about it's consistency whereas analytics does.  Our CreateIndex implementation in CNG is actually a compound operation of sending the create query, as well as doing various ensure calls.